common/io.go

```diff
@@ -0,0 +1,27 @@
+package common
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+// WriteFileAtomically create a temporary hidden file first
+// then move it into place. TempFile assigns mode 0600.
+func WriteFileAtomically(file string, content []byte) error {
```

Comment:
Is it common enough to put in this util repo?

Reply:
removed